lib/mcp.ts

```
@@ -0,0 +1,27 @@
+<title>Is Google ML Engineer Certification Worth It? (2025 ROI Analysis)</title>
+<meta_description>Comprehensive analysis of Google ML certification value, costs, and career impact based on 2025 market data.</meta_description>
+<introduction>...</introduction>
+<main_content>...</main_content>
+<schema_markup>...</schema_markup>
+      `,
+      url: 'https://example.com/generated-content'
+    }]
+  };
+}
+
+export interface McpToolResponse {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'McpToolResponse'. Consider to delete it or rename it.

'McpToolResponse' is defined here:


(lint/suspicious/noRedeclare)